go/vt/schemamanager/tablet_executor.go

```diff
@@ -31,9 +32,14 @@ import (
 	"vitess.io/vitess/go/vt/sqlparser"
 	"vitess.io/vitess/go/vt/topo"
 	"vitess.io/vitess/go/vt/vtctl/schematools"
+	"vitess.io/vitess/go/vt/vterrors"
 	"vitess.io/vitess/go/vt/vttablet/tmclient"
 )
 
+var (
+	lockRenewTimeout = 10 * time.Second // has to be a fraction of Topo's defaultLockTimeout (30sec at this time)
+)
```

Comment:
Should this be a flag, living in the same location as `remote_operation_timeout`. We allow users to tweek the remote-operation timeout which is what is used in lock shard. This should either be set to a fraction of it or be a flag too.

Reply:
Removed the variable, and set the rate limiter to use `*topo.RemoteOperationTimeout / 4`

Reply:
Thankyou! This change is awesome ❤️. I'll for sure use this refreshing of the lease in VTOrc.